src/examples/chrome/SheetSize.tsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Sheet } from '@zendeskgarden/react-chrome';
+import { Grid, Row, Col } from '@zendeskgarden/react-grid';
+import { Button } from '@zendeskgarden/react-buttons';
+import { Fieldset, Field, Radio, Label } from '@zendeskgarden/react-forms';
+
+const [col4, col6, px380, px480] = ['4', '6', '380px', '480px'];
```

Comment:
It feels like all the following codez would read simpler if the mapping could somehow be expressed up front as a constant. The behavior should be...

Col `size={4}` is Sheet `size="100%"`
Col `size={6}` is Sheet `size="100%"`
Sheet `size="380px"` is Col `size="auto"`
Sheet `size="480px"` is Col `size="auto"`